modules/apps/27-interchain-accounts/host/ibc_module_test.go

```diff
@@ -400,14 +400,16 @@ func (suite *InterchainAccountsTestSuite) TestOnRecvPacket() {
 		name          string
 		malleate      func()
 		expAckSuccess bool
+		eventErrorMsg string
 	}{
 		{
-			"success", func() {}, true,
+			"success", func() {}, true, "",
 		},
 		{
 			"host submodule disabled", func() {
 				suite.chainB.GetSimApp().ICAHostKeeper.SetParams(suite.chainB.GetContext(), types.NewParams(false, []string{}))
 			}, false,
+			"", // events are not emitted when the host submodule is disabled.
```

Comment:
I was thinking, is this actually desired behaviour? This could be useful debugging information via events.

Reply:
Yeah, I don't see any harm on maybe emitting the event with that information...